doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='githooks-prepare-proc-{context}']
+= Preparing Git hooks 
+
+In an authoring environment you can use Git hooks to execute custom operations when the source code of a project in {CENTRAL} is changed. The typical use of Git hooks is interaction with an upstream repository. 
```

Comment:
"The typical use of Git hooks is interaction with an upstream repository. "

Would it be correct to say: The typical use of Git hooks is for interaction with an upstream repository.